docs/articles/vs-test-adapter/Known-Problems.md

```diff
@@ -1,24 +1,30 @@
 # Known Problems
 
-1. There is no test status in Visual Studio corresponding to NUnit's Inconclusive result, so tests with this result are reported as Not Run. Click on the individual test to see the result.
+1. Using both the VSIX adapter and nuget adapters at the same time can lead to discovery issues, ref [Issue769](https://github.com/nunit/nunit3-vs-adapter/issues/769).  The resolution is to uninstall the VSIX adapter, and only use the nuget adapters.  The VSIX adapters are being deprecated in later VS versions.
 
-2. Theories are reported as individual cases, rather as a single unit.
+2. Support for Explicit keyword
 
-3. In NUnit, tests have names, which are not necessarily unique. Visual Studio wants the names to be unique. So if two tests have the same name, VS displays a warning message in the output window. The message may be ignored. Two separate results will be shown under the single test in the explorer pane.
+    1 Adapter versions in the 3.XX series:  From version 16.2 (Visual Studio 2019) the Explicit keyword is no longer explicitly supported by Visual Studio.  If you want to use it, it does exist as a Category also, so you can use it as a category filter.
 
-4. Startup performance is substantially improved but is still slower than using NUnit directly.
+    2 Adapter versions in the 4.XX series support the explicit keyword.
 
-5. Applies to pre-VS2017:  A VSIX adapter of older version will be used regardless of version of NuGet adapter.
+3. There is no test status in Visual Studio corresponding to NUnit's Inconclusive result, so tests with this result are reported as Not Run. Click on the individual test to see the result.
```

Comment:
```suggestion
3. There is no test status in Visual Studio corresponding to NUnit's `Inconclusive` result, so tests with this result are reported as `Not Run`. Click on the individual test to see the result.
```

Reply:
Nitpick: "Not Run" is not code, it's part of the reported result that is output. I'd use quotes.